recipes-bsp/u-boot/u-boot-harden.inc

```diff
@@ -17,7 +17,6 @@ SRC_URI:append:use-mainline-bsp = "\
     file://0003-toradex-integrate-command-whitelisting-upstream.patch \
     file://0004-toradex-integrate-bootm-protection-upstream.patch \
     file://0005-toradex-integrate-CLI-access-protection-upstream.patch \
-    file://0006-arm-dts-add-missing-newline-to-eof-upstream.patch \
```

Comment:
Actually the reason for having the gap is to keep the upstream and downstream patches in sync while avoiding the duplication of the patches that are common between upstream and downstream. To clarify, here's a table with the patches for both cases:

```
|-----------------------------------------------------------------+-------------------------------------------------------------|
| Downstream patch                                                | Upstream patch                                              |
|-----------------------------------------------------------------+-------------------------------------------------------------|
| 0001-toradex-common-add-command-whitelisting-modules.patch      | same as upstream                                            |
| 0002-toradex-dts-add-fragment-file-to-configure-secure-bo.patch | same as upstream                                            |
| 0003-toradex-integrate-command-whitelisting-downstream.patch    | 0003-toradex-integrate-command-whitelisting-upstream.patch  |
| 0004-toradex-integrate-bootm-protection-downstream.patch        | 0004-toradex-integrate-bootm-protection-upstream.patch      |
| 0005-toradex-integrate-CLI-access-protection-downstream.patch   | 0005-toradex-integrate-CLI-access-protection-upstream.patch |
| 0006-arm-dts-add-missing-newline-to-eof-downstream.patch        | none                                                        |
| 0007-toradex-add-implementation-of-bootargs-protection.patch    | same as upstream                                            |
| 0008-toradex-integrate-bootargs-protection-downstream.patch     | 0008-toradex-integrate-bootargs-protection-upstream.patch   |
|-----------------------------------------------------------------+-------------------------------------------------------------|
```

Notice the gap at index 0006 is there because there's a patch that only exists in the downstream case. Also worth pointing out that the patches that are common are the bigger ones introducing actual code for new features while the ones that are not common are very small as they are responsible for the integration into U-Boot at few touch points.

Having said that, it would be possible to consider this `0006-arm-dts-add-missing-newline-to-eof-downstream.patch` as being part of a different patch set to remove the gap from the upstream patch set (that file would become a separate `0001-arm-dts-add-missing-newline-to-eof-downstream.patch` only applied on the downstream case). Would that look better in your opinion? If so, I would be okay to make that change since I don't have a strong opinion on it.

Reply:
@sergioprado Btw, I'm looking forward for the test results on the BSP images.

Reply:
Thanks for the clarification. I also don't have a strong opinion on this, and it seems the effort is not worth it, so lets leave it as is.